web_validation_dialog/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# See LICENSE file for full copyright and licensing details.
```

Comment:
We usually use short headers for the copyright and license (with link)

Reply:
Done. Thanks for the review.